CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to Auth0 projects
```

Comment:
@benkoller Do we need to write Auth0 projects here?

Reply:
Nope. I did not proofread thoroughly enough. Will resolve with the next commit.